<launch>
  <node name="button_checker"        pkg="aruco_strategy" type="button_checker_cond"/>
  <node name="mb1_action"
  <node name="strategy_control_loop" pkg="aruco_strategy" type="aruco_strategy" />
</launch>
